global.less

```diff
@@ -43,19 +43,4 @@
     max-width: 800px;
     width: 100%;
     margin: 0 auto;
-<<<<<<< HEAD
-=======
-  }
-  
-  .grow {
-    flex-grow: 1;
-  }
-  
-  .flex{
-    display: flex; 
-    @media (max-width: 500px) {
-    flex-direction: column; 
-    align-items: center; 
-    }
```

Comment:
Hey Tammy 

If you delete this, the home page is not responsive to mobile (500px)